src/packer.pkr.hcl

```
@@ -46,89 +46,98 @@ variable "skip_create_ami" {
   type        = bool
 }
 
-data "amazon-ami" "debian_bullseye" {
+data "amazon-ami" "windows" {
```

In keeping with our scheme of including the OS version name here, I suggest this:
```suggestion
data "amazon-ami" "windows_server_2022" {
```

---

Changed in https://github.com/cisagov/windows-commando-packer/pull/10/commits/dc820cfab375d9150fdd8042188bb5be993518e2.